src/Modal/Modal.jsx

```diff
@@ -65,8 +74,10 @@ Modal.propTypes = {
   title: React.PropTypes.string.isRequired,
   closeModal: React.PropTypes.func.isRequired,
   children: React.PropTypes.node.isRequired,
+  isFocusTrap: React.PropTypes.bool,
```

Comment:
nit: s/`isFocusTrap`/`focusTrapped`/ or `focusLocked`?
since usage will usually look like `<Modal title="foo" focusLocked>...</Modal>`

Reply:
Good point. I like the latter, so let's go with that.